app/src/components/staking/DelegationsOverview.vue

```diff
@@ -52,6 +57,14 @@ export default {
   computed: {
     ...mapState([`session`]),
     ...mapGetters([`address`, `network`, `networks`]),
+    stakedBalance() {
+      const liquidBalance =
+        Number(this.balances[0].total) - Number(this.balances[0].available)
```

Comment:
this needs to be filtered by the stakingDenom not just taking the first balance

Reply:
true